sources/packages/backend/apps/api/src/route-controllers/application/application.controller.service.ts

```diff
@@ -334,4 +337,41 @@ export class ApplicationControllerService {
 
     return supportingUserDetails;
   }
+
+  /**
+   * Process application in calculation details.
+   * @param studentId student id.
+   * @param programYearId program year id.
+   * @param currentAssessmentId current assessment id.
+   * @return processed object.
+   */
+  async processApplicationInCalculationDetails(
```

Comment:
The idea of controller service is for having shared code(common methods) to be shared between multiple controllers. 

e.g. if there is a method `getApplication()` used by 

`application.students.controller` and
`application.aest.controller` then we move this  `getApplication()` into `application.controller.service`
so that the logic can be shared.